helm/metrics-server-app/values.yaml

```diff
@@ -9,6 +9,8 @@ k8sAppLabel: metrics-server
 
 replicas: 1
 
+e2e: false
```

Comment:
What happens if this is true?

Reply:
It creates the giantswarm-critical priority class. In production this is created with ignition.

With Helm 2 we called EnsureTillerInstalled which needed to do this so the Tiller pod would start.